infra/src/main/resources/db/migration/prod/V39__add_projektivelho_dictionaries.sql

```diff
@@ -0,0 +1,53 @@
+create table integrations.projektivelho_document_type
```

Comment:
Alkoi olla aika monta näitä pikkutauluja niin siirsin toiseen filuun. Kovasti näyttää myös siltä että tekisi mieli vaihtaa nämä kokonaan omaan skemaan, mutta sitten meillä olisi hassusti ratko integrations-schemassa ja projektivelho omassaan